### DISCUSSION TITLE
How to use it in the chrome extension service worker?

### DISCUSSION THREAD

**Discussion by Maksimka101** [simolus3/drift#3253] [Q&A]
I have the `TypeError: Failed to fetch` error when call the `WasmDatabase.open` method. The `WasmDatabase.probe` can find only in memory storage

However DB opens fine from content script or action script

**Comment by simolus3** [comment]
Do you see any stack traces to the source of the type error? I'm not familiar with developing Chrome extensions and don't know how easy it will be for me to reproduce the issue, so having more information would be helpful.
